fineract-provider/dependencies.gradle

```diff
@@ -208,9 +208,9 @@ dependencies {
     testImplementation( 'io.cucumber:cucumber-spring',
             'io.github.classgraph:classgraph',
             'io.cucumber:cucumber-core',
-            'io.cucumber:cucumber-java',
-            'io.cucumber:cucumber-java8',
-            'io.cucumber:cucumber-junit-platform-engine',
+            'io.cucumber:cucumber-java:7.16.1',
```

Comment:
I think the versions should only be set in the `org.apache.fineract.dependencies.gradle`, would it be possible to move all dependency versions to a common place?

Reply:
Done! updated